_posts/2016-12-18-from-blm-to-black-liberation.md

```diff
@@ -1,57 +0,0 @@
----
```

Comment:
This wasn't read for SF Feminist Book Club. Yesterday @lizhubertz and I decided to focus the site on only one book club since having two was making our design confusing and our anarchist group isn't open to the public. I'd be okay with keeping it in if no one minds it not being accessible from the site navigation for now, or we can add it back later if needed.